src/core/Table/Table.tsx

```diff
@@ -230,6 +249,7 @@ export const Table = <
     },
     useFlexLayout,
     useSortBy,
+    useExpanded,
```

Comment:
Just curious..Why decided to put this in the middle? :D

Reply:
I initially put it as last parameter. But this is the reason, I had to put it in the middle :D
![image](https://user-images.githubusercontent.com/36437897/117635423-7619c100-b188-11eb-9ad9-15e04bdafcf6.png)